ci/configs/deploy_with_ackama_ec2_capistrano.yml

```diff
@@ -2,6 +2,7 @@
 staging_hostname: "staging.example.com"
 production_hostname: "www.example.com"
 git_repo_url: ""
+use_typescript: false
```

Comment:
Prior to this change, only the opt-in variants were referencing `use_typescript`